inference-engine/src/mkldnn_plugin/mkldnn_plugin.cpp

```diff
@@ -183,6 +184,8 @@ static void TransformationUpToCPUSpecificOpSet(std::shared_ptr<ngraph::Function>
     }
     manager.register_pass<ngraph::pass::ConvertPrecision>(precisions);
 
+    manager.register_pass<ngraph::pass::ConvertConvertLike>();
```

Comment:
Is there any reason why this transformation is enabled only for CPU? 
I suppose other plugins may benefit from this transformation also.

Reply:
Otherwise the code of this transformation must be located inside plugin.